insights/parsers/tests/test_tomcat_virtual_dir_context.py

```diff
@@ -70,20 +70,20 @@ def test_tomcat_virtual_dir_context_not_found():
 
 def test_tomcat_virtual_dir_context_exceptions():
     for parser in [TomcatVirtualDirContextFallback, TomcatVirtualDirContextTargeted]:
-        with pytest.raises(SkipException) as excinfo:
-            parser(context_wrap(ERRORS_1))
-            assert 'VirtualDirContext not used.' in str(excinfo.value)
+        # with pytest.raises(SkipException) as excinfo:
+        #     parser(context_wrap(ERRORS_1))
+        #     assert 'VirtualDirContext not used.' in str(excinfo.value)
```

Comment:
I suggest just taking out commented code.  We can always restore from a previous version.

Reply:
Yea, was leaving it in for the formal walk through, removing.